react/week1/react-week1/src/App.js

```diff
@@ -0,0 +1,45 @@
+import './App.css';
+
+const todoList = [
+  {
+    id: 1,
+    description: "Get out of bed",
+    date: "Wed Sep 13 2021",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+    date: 'Thu Sep 14 2021',
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+    date: "Fri Sep 15 2021",
+  }
+];
+
+function TodoTableRow(props) {
+  return <li>
+    <p>{props.text}, {props.date}</p>
+  </li>;
+}
+
+function TodoTable() {
+  return <div  style={{ width: "300px", paddingLeft: "20px", paddingRight: "20px" }}>
+    <h1>ToDo List:</h1>
+    <ul>
+      {todoList.map(TodoItem => <TodoTableRow text={TodoItem.description} date={TodoItem.date} />)}
```

Comment:
Here the console is showing a warning that says Each child in a list should have a unique "key" prop.
you can avoid the problem by providing a unique key that has the value of TodoItem.id